### ISSUE TITLE
esling-plugin-vue not found, fixed by installing globally.

### ISSUE THREAD

**Issue by sarkasmus** [eslint/eslint#9839]
Hello,
I found out a similar bug to #3782 with eslint-plugin-vue. It had the same solution like this - to install eslint-plugin-vue globally.

**Tell us about your environment**

* **ESLint Version: 4.15.0**
* **Node Version: 8.7.0**
* **npm Version: 5.4.2**

**What parser (default, Babel-ESLint, etc.) are you using?**
I'm using default parser.

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

{
  "extends": "plugin:vue/essential",
  "parserOptions": {
    "ecmaVersion": 8,
    "sourceType": "module"
  },
  "rules": {
    "for-direction": "error",
    "getter-return": "error",
    "no-compare-neg-zero": "warn",
    "no-cond-assign": "error",
    "no-console": "warn",
    "no-constant-condition": "error",
    "no-control-regex": "warn",
    "no-debugger": "error",
    "no-dupe-args": "error",
    "no-dupe-keys": "error",
    "no-duplicate-case": "error",
    "no-empty": "error",
    "no-empty-character-class": "error",
    "no-ex-assign": "error",
    "no-extra-boolean-cast": "error",
    "no-extra-parens": "warn",
    "semi": [
      "error",
      "never"
    ],
    "no-func-assign": "error",
    "no-inner-declarations": "error",
    "no-invalid-regexp": "error",
    "no-irregular-whitespace": "error",
    "no-obj-calls": "error",
    "no-prototype-builtins": "error",
    "no-regex-spaces": "error",
    "no-sparse-arrays": "error",
    "no-template-curly-in-string": "warn",
    "no-unreachable": "error",
    "no-unsafe-finally": "error",
    "no-unsafe-negation": "error",
    "use-isnan": "error",
    //"valid-jsdoc": "warn",
    "valid-typeof": "error",

    // Best practices
    "accessor-pairs": "warn",
    "array-callback-return": "error",
    "block-scoped-var": "error",
    "class-methods-use-this": "off",
    "complexity": ["warn", 5],
    "consistent-return": "error",
    "curly": "error",
    "default-case": "error",
    "dot-location": ["error", "object"],
    "dot-notation": "error",
    "eqeqeq": "error",
    "guard-for-in": "error",
    "quotes": ["error", "single"],
    "no-alert": "error",
    "no-caller": "error",
    "no-fallthrough": "error",
    "no-case-declarations": "error",
    "no-div-regex": "error",
    "no-else-return": "error",
    "no-empty-function": "error",
    "no-empty-pattern": "error",
    "no-eq-null": "off",
    "no-eval": "error",
    "no-extend-native": "error",
    "no-extra-bind": "error",
    "no-invalid-this": "error",
    "no-labels": "error",
    "no-multi-spaces": "error",
    "no-multi-str": "error",
    "no-param-reassign": "error",
    "no-redeclare": "error",
    "no-useless-return": "error",
    "no-unused-vars": "error",
    "no-use-before-define": "error",
    "array-bracket-newline": ["error", "consistent"],
    "camelcase": "error",
    "capitalized-comments": "error",
    "comma-dangle": "error",
    "comma-spacing": ["error", {"after": true, "before": false}],
    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
    "eol-last": ["error", "always"],
    "func-call-spacing": ["error", "never"],
    "indent": ["error", 2],
    "linebreak-style": ["error", "unix"],
    "max-len": ["error", 120],
    "max-lines": ["error", 300],
    "max-statements-per-line": ["error", {"max": 1}],
    "new-cap": "error",
    "no-tabs": "error",
    "no-trailing-spaces": "off",
    "padding-line-between-statements": [
      "error",
      { "blankLine": "always", "prev": "*", "next": "return" },
      { "blankLine": "always", "prev": ["const", "let", "var", "expression"], "next": "*"},
      { "blankLine": "any", "prev": ["const", "let", "var", "expression"], "next": ["const", "let", "var", "expression"]},
      { "blankLine": "always", "prev": "import", "next": "*"},
      { "blankLine": "any", "prev": "import", "next": "import"}
    ],
    "require-jsdoc": ["error", {
      "require": {
        "FunctionDeclaration": true,
        "MethodDefinition": true,
        "ClassDeclaration": true,
        "ArrowFunctionExpression": false,
        "FunctionExpression": false
      }
    }],
    "vue/no-reserved-keys": "error"
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```html


<template src="./ExampleComponent.html"></template>
<script>
  export default {
    name: 'ExampleComponent',
    computed: {
      /**
       * Test no-reserved-keys
       * */
      $on () {
          
      }
  }
}

</script>


```

```bash
eslint src/component/ExampleComponent
```

**What did you expect to happen?**
I expected to get error of rule `no-reserved-keys`

**What actually happened? Please include the actual, raw output from ESLint.**
```
Cannot find module 'eslint-plugin-vue'
Referenced from: /home/michal/Plocha/corlex-photos-theme/.eslintrc
Error: Cannot find module 'eslint-plugin-vue'
Referenced from: /home/michal/Plocha/corlex-photos-theme/.eslintrc
    at ModuleResolver.resolve (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/util/module-resolver.js:74:19)
    at resolve (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:466:29)
    at load (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:542:26)
    at configExtends.reduceRight (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:421:36)
    at Array.reduceRight (<anonymous>)
    at applyExtends (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:403:28)
    at loadFromDisk (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:514:22)
    at Object.load (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config/config-file.js:550:20)
    at Config.getLocalConfigHierarchy (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config.js:228:44)
    at Config.getConfigHierarchy (/home/michal/.nvm/versions/node/v8.7.0/lib/node_modules/eslint/lib/config.js:180:43)

```

**Comment by ilyavolodin** [comment]
This is intended behavior. You are executing ESLint by running `eslint src/component/ExampleComponent` which, based on NPM rules, will always execute globally installed version of ESLint (unless you used symlinks). Global version of ESLint requires all plugins to also be installed globally. You can fix it by executing local version of ESLint with the following command: `node_modules/.bin/eslint src/component/ExampleComponent`.

**Comment by j-f1** [comment]
If you have npm v5 or above, you can also do `npx eslint src/component/ExampleComponent`.

**Comment by sarkasmus** [comment]
Ok, thanks for explaining. Should I close the issue now?

**Comment by ilyavolodin** [comment]
Looks like the issue is resolved. Closing.
